ambassador/ambassador/ir/irauth.py

```diff
@@ -165,11 +143,21 @@ def config_dict(self):
             "cluster": self.cluster.name
         }
 
-        for key in [ 'allowed_headers', 'path_prefix', 'timeout_ms', 'weight' ]:
+        for key in [ 'allowed_resquest_headers', 'allowed_authorization_headers', 'path_prefix', 'timeout_ms', 'weight' ]:
```

Comment:
`allowed_resquest_headers` is misspelled here. Hmm. :smile:

Reply:
I will fix the type, but it makes me think what `config_dict(self)` does. Shouldn't this have failed? I need to look into that.

Reply:
I feel like it should have? That's really strange.